src/test/java/com/kainos/ea/integration/WebServiceIntegrationTest.java

```diff
@@ -30,4 +32,18 @@ void getEmployees_shouldReturnListOfEmployees() {
         assertTrue(response.size() > 0);
     }
 
-}
+    @Test
+    void getJobRoles_shouldReturnListOfJobRoles_withCapabilityNotNull() {
+
+        ObjectMapper mapper = new ObjectMapper();
+        JsonNode response = APP.client().target("http://localhost:8080/api/job-roles")
+                .request()
+                .get(JsonNode.class);
+        List<JobRole> JobRoleList = mapper.convertValue(
+                response,
+                new TypeReference<List<JobRole>>(){}
+        );
+
+        assertNotNull(JobRoleList.get(0).getCapability_name());
```

Comment:
can you add this check to an existing integration test so we reduce the amount of HTTP calls needing to be made

Reply:
added not null checks for each field to existing test